en/release-notes.md

```diff
@@ -25,6 +25,96 @@ The official Python client library for MAAS is available in the Ubuntu 18.04
+
+     *  When saving node objects (e.g. making any update of a machine, device, rack controller, etc), MAAS validated changes across various fields. This required an increased number of queries for fields, even when they were not being updated. MAAS now tracks specific fields that change and only performs queries for those fields.
+     Example: To update a power state, MAAS would perform 11 queries. After these improvements, , only 1 query is now performed.
+
+     * On every transaction, MAAS performed 2 queries to update the timestamp. This has now been consolidated into a single query per transaction.
+
+   These changes  greatly improve MAAS performance and database utilisation in larger environments. More improvements will continue to be made as we continue to examine various areas in MAAS.
+
+
+ - **UI optimisations**
+   MAAS is now being optimised to reduce the amount of data loaded in the websocket API to render the UI. This is targeted at only processing data for viewable information, improving various legacy areas. Currently, the work done in this area includes:
+
+   Script results are only loaded for viewable nodes in the machine listing page, reducing the overall amount of data loaded.
+
+   The node object is updated in the websocket only when something has changed in the database, reducing the data transferred to the clients as well as the amount of internal queries.
+
+#### Audit logging
+
+Continuing with the audit logging improvements, alpha2 now adds audit logging for all
+user actions that affect Hardware Testing & Commissioning.
+
+#### KVM pod improvements
+
+MAAS’ KVM pods was initially developed as a feature to help developers quickly iterate
+and test new functionality while developing MAAS. This, however, because a feature that
+allow not only developers, but also administrators to make better use of resources across
+their datacenter. Since the feature was initially create for developers, some features
+were lacking. As such, in 2.4 we are improving the usability of KVM pods:
+
+
+ - **Pod AZ’s**
+   MAAS now allows setting the physical zone for the pod. This helps administrators by conceptually placing their KVM pods in a AZ, which enables them to request/allocate machines on demand based on its AZ. All VM’s created from a pod will inherit the AZ.
+
+
+ - **Pod tagging**
+   MAAS now adds the ability to set tags for a pod. This allows administrators to use tags to allow/prevent the creation of a VM inside the pod using tags. For example, if the administrator would like a machine with a ‘tag’ named ‘virtual’, MAAS will filter all physical machines and only consider other VM’s or a KVM pod for machine allocation.
+
+### Bug fixes
+
+Please refer to the following for all bug fixes in this release.
+
+https://launchpad.net/maas/+milestone/2.4.0alpha2
```

Comment:
Will this be rendered as a link automatically?

Reply:
yes!